### PR TITLE
test: bump jasmine to 4.x to get ArrayBuffer equality matchers

### PR DESCRIPTION
**Pull request by fippo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,10 +47,10 @@
     "core-js": "3.19.1",
     "eslint": "8.1.0",
     "eslint-plugin-import": "2.25.2",
-    "jasmine-core": "3.5.0",
+    "jasmine-core": "^4.2.0",
```
Comment (saghul) [inline package.json:50]: can you ping it to the exact version pl?

File: package.json [hunk 2]
```diff
@@ -47,10 +47,10 @@
...
     "karma": "6.3.16",
     "karma-chrome-launcher": "3.1.0",
-    "karma-jasmine": "3.1.1",
+    "karma-jasmine": "^4.0.2",
```
Comment (saghul) [inline package.json:53]: ditto

**Review by saghul — CHANGES_REQUESTED** [review]

**Comment by NewEraCracker** [comment]
Hello, while you are at it. Probably a good idea to update the following related packages - I have tested the following versions:

```
    "karma": "6.3.20",
    "karma-chrome-launcher": "3.1.1",
    "karma-jasmine": "5.0.1",
    "karma-sourcemap-loader": "0.3.8",
````

My two cents.

**Comment by fippo** [comment]
leaving karma-jasmine at 4.x for now

**Review by nils-ohlmeier — APPROVED** [review]

**Pull request merged**